regression.py

```diff
@@ -1571,7 +1574,7 @@ def testirintens(self):
 
     "GAMESS/WinGAMESS/dvb_td_2007.03.24.r1.out":    GAMESSUSTDDFTTest,
 
-    "Gaussian/Gaussian09/dvb_gopt_revA.02.out":         GaussianGeoOptTest,
+    "Gaussian/Gaussian09/dvb_gopt_revA.02.out":         GenericGeoOptTest,

```

Comment:
I don't think this change should have been made here. ```GaussianGeoOptTest``` does exist and should have been imported at the same time as ```GenericGeoOptTest``` from ```test/data/testGeoOpt.py```.